### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Nezerwa**
>In this project i
-Added a portfolio version that is compatible with the desktop screens
-Added media queries

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,11 +84,48 @@ <h2 class="tonic">Tonic</h2>
                         <li>javascript</li>
                     </ul>
                     <button class="see-project">See Project</button>
+                  </div>
+        </section> 
+        <section class="sec2 desktop">
+          <div class="sec-content">
+            <div class="image-div"> 
+              <img src="images/snapshoot Portfolio.png" alt="image" class="workplace-image">
```
Comment (christianbiring1) [inline index.html:92]: - [ ] Kindly consider using the correct path for your image.

Your design :point_down: 
image

File: index.html [hunk 2]
```diff
@@ -4,36 +4,39 @@
...
             </h1>
                 <p class="p-intro">
-                    I’m a software developer! I can help you build a product , 
-                    feature or website Look through 
-                    some of my work and experience! If you like what you see and have a project you need
+                    I’m a software developer! I can help you build a product ,
+                    feature or website Look through<br />
+                    some of my work and experience! If you like what you see and have a project you need<br />
                      coded, don’t hesitate to contact me.
                 </p>
```
Comment (christianbiring1) [inline index.html:41]: - [ ] Kindly make sure you stick to the design as much as you can, You may consider using the same typo(font) for your text
Your design :point_down: 
image


Figma template :point_down: 
image

File: styles.css [hunk 3]
```diff
@@ -14,32 +14,20 @@ body {
 }
 
 main {
-  background-color: #e5e5e5;
+  background-color: hsl(0, 0%, 90%);
   width: 100%;
 }
```
Comment (christianbiring1) [inline styles.css]: - [ ] Kindly make sure you stick to the design as much as possible. You may consider fixing the introduction section of your page
- [ ] Also make sure your page does not overflow on the mobile version. we do not need to scroll to the whole page content. Kindly fix that.

Your design :point_down: 
image

image
image


Figma template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Nezerwa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] Good commit history 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- in your README in the get started section, You may consider providing some commands to get the project run locally.
`hint: You may say: to clone the repos : in the terminal run git clone <REPOS_LINK>`
`cd <DIRECTORY_NAME>`
`...`
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -140,10 +219,47 @@ <h2 class="tonic">Tonic</h2>
...
+                   <ul class="list-tech f-margin">
+                       <li>html</li>
+                       <li>Ruby on rails</li>
+                       <li>css</li>
+                       <li>javascript</li>
+                   </ul>
+                   <button class="see-project blue b-margin">See Project</button>
+               </div>
+               </div>
             </section>
```
Comment (levy002) [inline index.html:256]: - [OPTIONAL] I have noticed that you are creating a new desktop card for the desktop but the main purpose of using media queries is to adapt our website to different screens by creating different styles for the same Html file without duplicating it. I highly recommend you not create a new project card for the desktop but to use CSS to manipulate styles and how it must appear on the different screens without changing your Html file entirely. :wink: Please to d all that on all project cards.

File: index.html [hunk 5]
```diff
@@ -140,10 +219,47 @@ <h2 class="tonic">Tonic</h2>
...
-            <section class="sec5">
+            <section class="sec5 mobile">
                 <div class="sec-content">
+                  <div class="image-div"> 
                     <img src="images/professional.png" alt="image" class="workplace-image">
+                  </div>
```
Comment (levy002) [inline index.html:261]: - [OPTIONAL] Kindly make sure to fit your project image into the project card correctly in order to meet the Figma design. :wink: 

- [OPTIONAL] Kindly make sure that the `See Project button` looks exactly as the one on the provided Figma design:wink
-
    - ### Your design
    image


    - ### Figma design
   image

**Review by levy002 — APPROVED** [review]
## Hello @Nezerwa 

congratulations on working on the suggested changes.  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure to align correctly the `ABOUT MYSELF` section in order to meet the provided Figma design :wink: 

    - ### Your design
    image


   - ### Figma design
     image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**